conans/client/tools/env.py

```diff
@@ -65,6 +69,7 @@ def _environment_add(env_vars, post=False):
                     apply_vars[name] = old + os.pathsep + apply_vars[name]
                 else:
                     apply_vars[name] += os.pathsep + old
+            apply_vars[name] = os.pathsep.join(_unique(apply_vars[name].split(os.pathsep)))
```

Comment:
It seems that it can be done in place with a more common python idiom like:

```python
items = apply_vars[name].split(os.pathsep)
apply_vars[name] = os.pathsep.join(OrderedDict.fromkeys(items))
```

Reply:
> 
> 
> It seems that it can be done in place with a more common python idiom like:
> 
> ```python
> items = apply_vars[name].split(os.pathsep)
> apply_vars[name] = os.pathsep.join(OrderedDict.fromkeys(items))
> ```

Yes, indeed. I was under the impression that `OrderedDict` was the Python3-only thing but actually it's included in Python 2.7 as well.

Reply:
Great. If you could manage to add some test (ask for help if necessary), this can be merged for next Conan 1.31.

Reply:
> Could you please add a test that will cover this case? Ask for guidance if it is not evident how/where.

Yes, please. I'm not familiar with Conan testing framework. Is  https://github.com/conan-io/conan/blob/4f9822deb2ef4fc7cd8b50a5091beff831dd546e/conans/test/unittests/client/tools/win/vcvars_test.py#L60 the correct place for these additional tests? And how to run Conan tests locally?

Reply:
It seems that the ``VirtualEnvGeneratorTest`` in conans/test/unittests/client/generators/virtualenv_test.py would be a good place, adding another variable with repeated paths.

Locally, you can run it with: ``nosetests conans/test/unittests/client/generators/virtualenv_test.py``. You might need to ``pip install -r requirements_dev.txt`` and ``pip install -r requirements_server.txt`` if missing deps. Please let me know if this helps, otherwise I can contribute the test myself. Thanks again for the contribution!

Reply:
> Please let me know if this helps, otherwise I can contribute the test myself.

Looks like sorting it out needs some time but right now I'm a little bit busy. If you can write tests yourself with little efforts please do!